exporter/collectors/test_collector.py

```diff
@@ -47,7 +61,9 @@ def parse_test_details_response(r):
 
 class TestCollector(object):
 
-    def __init__(self, username, api_key, tags):
+    def __init__(self, use_v1_uptime_endpoints, use_v1_maintenance_windows_endpoints, username, api_key, tags):
```

Comment:
consider using optional arguments here.

Per my last comment:

```suggestion
    def __init__(self, use_v1_uptime_endpoints=False, use_v1_maintenance_windows_endpoints=False, username, api_key, tags):
```

A populated string will eval to True 

```
MY_STR = "Test"

bool(MY_STR)

True
```

For additional safety you could cast to bool on line 65 and 66:

```
        self.use_v1_uptime_endpoints = bool(use_v1_uptime_endpoints)
        self.use_v1_maintenance_windows_endpoints = bool(use_v1_maintenance_windows_endpoints)
```